src/components/Like.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState } from 'react';
+import { patchReview } from '../utils/utils'
+
+
+
+function Like({vote, setVote, review_id, voted, setVoted}) {
+    
+
+    function handleLike () {
+        if(voted){
+            setVoted(false);
+            setVote(vote - 1) 
```

Comment:
Here once the user clicks the hand again after a 'like' , the votes are decreased on the page, but they are not updated in the database. Please make this consistent. At the moment after pressing the like button 10 times starting with 5 like, the page shows still 5, but after a refresh we see 15.